Makefile

```diff
@@ -130,6 +130,18 @@ test: all
 	@echo
 	@echo "Testing Red Alert mod MiniYAML..."
 	@./utility.sh ra --check-yaml
+	@echo
+	@echo "Testing TS mod content..."
+	@./utility.sh ts-content --check-yaml
+	@echo
+	@echo "Testing Dune 2000 mod content..."
```

Comment:
IMO it would be cleaner if these were included with their mods above:
```
Testing Dune 2000 mod MiniYAML...
Testing mod: Mod Content Manager
Testing Fluent references
Testing mod: Dune 2000
Testing default sequences for ARRAKIS
Testing Fluent references
Testing map: Arakeen
Testing map: Atreides 01a
Testing map: Atreides 01b
...
```

Reply:
sure